compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1497,6 +1497,48 @@ static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
       entryPointFn, fftOp, tileSizes, DispatchLoweringPassPipeline::CPUDefault);
 }
 
+/// Sets the lowering configuration for dispatch region for
+/// linalg_ext.winograd.input_transform root op.
+static LogicalResult
+setRootConfig(mlir::FunctionOpInterface entryPointFn,
+              IREE::LinalgExt::WinogradInputTransformOp inputOp) {
+  assert(!getLoweringConfig(inputOp) && "expected lowering_config is not set");
+  auto iterationRank = inputOp.getIterationDomainRank();
+  SmallVector<int64_t> vecSizeHints(iterationRank, 1);
+  DistributionHeuristicConfig distConfig;
+  distConfig.vectorSizeHints = vecSizeHints;
+  SmallVector<int64_t> distTileSizes =
+      getDefaultDistributedLevelTileSizes(inputOp, distConfig);
+  TileSizesListType tileSizes;
+  tileSizes.push_back(distTileSizes);
+  SmallVector<int64_t> vecTileSizes(iterationRank, 1);
+  tileSizes.push_back(vecTileSizes);
+  return setOpConfigAndEntryPointFnTranslation(
+      entryPointFn, inputOp, tileSizes,
+      DispatchLoweringPassPipeline::CPULinalgExtTileAndVectorize);
+}
+
+/// Sets the lowering configuration for dispatch region for
+/// linalg_ext.winograd.input_transform root op.
+static LogicalResult
+setRootConfig(mlir::FunctionOpInterface entryPointFn,
```

Comment:
These two functions are almost identical. Can we combine them into a single function? Maybe using a template?

Reply:
I already have this on my next PR (not out yet) that adds the filter transform op, so I'll just wait until then for this. I'll send it out once this one lands.